package.json

```diff
@@ -175,35 +175,19 @@
   },
   "peerDependencies": {
     "@inertiajs/vue3": "^2.0.7",
-    "joi": "^17.13.0",
     "superstruct": "^2.0.0",
     "typescript": "^5.6.3",
-    "valibot": "^1.0.0",
-    "vue-router": "^4.5.0",
-    "yup": "^1.6.0",
-    "zod": "^3.24.0 || ^4.0.0"
+    "vue-router": "^4.5.0"
   },
   "peerDependenciesMeta": {
     "@inertiajs/vue3": {
       "optional": true
     },
-    "joi": {
-      "optional": true
-    },
-    "valibot": {
-      "optional": true
-    },
     "superstruct": {
       "optional": true
     },
     "vue-router": {
       "optional": true
-    },
-    "yup": {
-      "optional": true
-    },
-    "zod": {
```

Comment:
@romhml Wasn't there a reason we kept those in `peerDependencies` even after the standard-schema integration?

Reply:
These peer deps were already supported (without standard-schema), so it's just a hint when the user upgrades that he should also upgrade his validation library if its incompatible. We don't need to make this for other standard schema compatible libs, because they were never supported in the first place.

Reply:
The runtime error could be enough, but I don't see how keeping the peerDeps hurts